### ISSUE TITLE
Issue concerning evaluate network issue and beyond

### ISSUE THREAD

**Issue by koronfelina** [DeepLabCut/DeepLabCut#1765]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

So I re-trained the network from scratch and also through refine labels and I can't evaluate the network. I could create, analyze and label the videos before, but after re-training, I can't do anything past the "evaluate network" step. I have no idea where the issue is and I get a very long message, please see attached code.

### Operating System

operating system: windows 10

### DeepLabCut version

dlc version: I don't know. I get this message for both commands: 'import' is not recognized as an internal or external command, operable program or batch file.

### DeepLabCut mode

single animal

### Device type

gpu: NVIDIA GeForce RTX 2080 Ti

### Relevant log output

```shell
Original stack trace for 'save/RestoreV2':
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\__main__.py", line 22, in <module>
    deeplabcut.launch_dlc()
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\launch_script.py", line 47, in launch_dlc
    app.MainLoop()
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\wx\core.py", line 2167, in MainLoop
    rv = wx.PyApp.MainLoop(self)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\refine_labels.py", line 158, in refine_labels
    deeplabcut.refine_labels(self.config)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\refine_labels.py", line 55, in refine_labels
    refinement.show(config)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\refinement.py", line 897, in show
    app.MainLoop()
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\wx\core.py", line 2167, in MainLoop
    rv = wx.PyApp.MainLoop(self)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\evaluate_network.py", line 270, in evaluate_network
    deeplabcut.evaluate_network(
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\evaluate.py", line 769, in evaluate_network
    sess, inputs, outputs = predict.setup_pose_prediction(dlc_cfg)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\predict.py", line 44, in setup_pose_prediction
    restorer = tf.compat.v1.train.Saver()
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\training\saver.py", line 923, in __init__
    self.build()
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\training\saver.py", line 935, in build
    self._build(self._filename, build_save=True, build_restore=True)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\training\saver.py", line 963, in _build
    self.saver_def = self._builder._build_internal(  # pylint: disable=protected-access
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\training\saver.py", line 533, in _build_internal
    restore_op = self._AddRestoreOps(filename_tensor, saveables,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\training\saver.py", line 353, in _AddRestoreOps
    all_tensors = self.bulk_restore(filename_tensor, saveables, preferred_shard,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\training\saver.py", line 601, in bulk_restore
    return io_ops.restore_v2(filename_tensor, names, slices, dtypes)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\ops\gen_io_ops.py", line 1500, in restore_v2
    _, _, _op, _outputs = _op_def_library._apply_op_helper(
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 744, in _apply_op_helper
    op = g._create_op_internal(op_type_name, inputs, dtypes=None,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\framework\ops.py", line 3697, in _create_op_internal
    ret = Operation(
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\framework\ops.py", line 2101, in __init__
    self._traceback = tf_stack.extract_stack_for_node(self._c_op)

Config:
{'all_joints': [[0], [1], [2], [3]],
 'all_joints_names': ['bodypart1', 'bodypart2', 'bodypart3', 'objectA'],
 'batch_size': 1,
 'crop_pad': 0,
 'dataset': 'training-datasets\\iteration-0\\UnaugmentedDataSet_EyeBehaviour_B19N3Feb10\\EyeBehaviour_B19N3_LMK95shuffle1.mat',
 'dataset_type': 'imgaug',
 'deterministic': False,
 'fg_fraction': 0.25,
 'global_scale': 0.8,
 'init_weights': 'C:\\Users\\81904\\anaconda3\\envs\\deeplabcut\\lib\\site-packages\\deeplabcut\\pose_estimation_tensorflow\\models\\pretrained\\resnet_v1_101.ckpt',
 'intermediate_supervision': False,
 'intermediate_supervision_layer': 12,
 'location_refinement': True,
 'locref_huber_loss': True,
 'locref_loss_weight': 1.0,
 'locref_stdev': 7.2801,
 'log_dir': 'log',
 'mean_pixel': [123.68, 116.779, 103.939],
 'mirror': False,
 'net_type': 'resnet_101',
 'num_joints': 4,
 'optimizer': 'sgd',
 'pairwise_huber_loss': True,
 'pairwise_predict': False,
 'partaffinityfield_predict': False,
 'regularize': False,
 'scoremap_dir': 'test',
 'shuffle': True,
 'snapshot_prefix': 'F:\\Eye\\DeepLabCut\\USE_FROM_DRIVED\\DeepLabCut_NAMEOK_new\\EyeBehaviour_B19N3-LMK-2022-02-10\\dlc-models\\iteration-0\\EyeBehaviour_B19N3Feb10-trainset95shuffle1\\test\\snapshot',
 'stride': 8.0,
 'weigh_negatives': False,
 'weigh_only_present_joints': False,
 'weigh_part_predictions': False,
 'weight_decay': 0.0001}
Using snapshot-110000 for model F:\Eye\DeepLabCut\USE_FROM_DRIVED\DeepLabCut_NAMEOK_new\EyeBehaviour_B19N3-LMK-2022-02-10\dlc-models\iteration-0\EyeBehaviour_B19N3Feb10-trainset95shuffle1
2022-04-08 20:28:21.600005: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:0 with 8962 MB memory:  -> device: 0, name: NVIDIA GeForce RTX 2080 Ti, pci bus id: 0000:01:00.0, compute capability: 7.5
2022-04-08 20:28:21.600250: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1525] Created device /job:localhost/replica:0/task:0/device:GPU:1 with 8962 MB memory:  -> device: 1, name: NVIDIA GeForce RTX 2080 Ti, pci bus id: 0000:02:00.0, compute capability: 7.5
2022-04-08 20:28:22.742067: W tensorflow/core/framework/op_kernel.cc:1745] OP_REQUIRES failed at save_restore_v2_ops.cc:207 : NOT_FOUND: Key resnet_v1_101/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
Traceback (most recent call last):
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\client\session.py", line 1380, in _do_call
    return fn(*args)
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\client\session.py", line 1363, in _run_fn
    return self._call_tf_sessionrun(options, feed_dict, fetch_list,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\client\session.py", line 1456, in _call_tf_sessionrun
    return tf_session.TF_SessionRun_wrapper(self._session, options, feed_dict,
tensorflow.python.framework.errors_impl.NotFoundError: 2 root error(s) found.
  (0) NOT_FOUND: Key resnet_v1_101/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
         [[{{node save/RestoreV2}}]]
         [[save/RestoreV2/_335]]
  (1) NOT_FOUND: Key resnet_v1_101/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
         [[{{node save/RestoreV2}}]]
0 successful operations.
0 derived errors ignored.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\training\saver.py", line 1404, in restore
    sess.run(self.saver_def.restore_op_name,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\client\session.py", line 970, in run
    result = self._run(None, fetches, feed_dict, options_ptr,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\client\session.py", line 1193, in _run
    results = self._do_run(handle, final_targets, final_fetches,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\client\session.py", line 1373, in _do_run
    return self._do_call(_run_fn, feeds, fetches, targets, options,
  File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\tensorflow\python\client\session.py", line 1399, in _do_call
    raise type(e)(node_def, op, message)  # pylint: disable=no-value-for-parameter
tensorflow.python.framework.errors_impl.NotFoundError: 2 root error(s) found.
  (0) NOT_FOUND: Key resnet_v1_101/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
         [[node save/RestoreV2
 (defined at C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\predict.py:213)
]]
         [[save/RestoreV2/_335]]
  (1) NOT_FOUND: Key resnet_v1_101/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
         [[node save/RestoreV2
 (defined at C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\predict.py:213)
]]
0 successful operations.
0 derived errors ignored.

Errors may have originated from an input operation.
Input Source operations connected to node save/RestoreV2:
In[0] save/Const:
In[1] save/RestoreV2/tensor_names:
In[2] save/RestoreV2/shape_and_slices:

Operation defined at: (most recent call last)
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\runpy.py", line 194, in _run_module_as_main
>>>     return _run_code(code, main_globals, None,
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\runpy.py", line 87, in _run_code
>>>     exec(code, run_globals)
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\__main__.py", line 22, in <module>
>>>     deeplabcut.launch_dlc()
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\launch_script.py", line 47, in launch_dlc
>>>     app.MainLoop()
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\wx\core.py", line 2167, in MainLoop
>>>     rv = wx.PyApp.MainLoop(self)
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\refine_labels.py", line 158, in refine_labels
>>>     deeplabcut.refine_labels(self.config)
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\refine_labels.py", line 55, in refine_labels
>>>     refinement.show(config)
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\refinement.py", line 897, in show
>>>     app.MainLoop()
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\wx\core.py", line 2167, in MainLoop
>>>     rv = wx.PyApp.MainLoop(self)
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\gui\analyze_videos.py", line 501, in analyze_videos
>>>     scorername = deeplabcut.analyze_videos(
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\pose_estimation_tensorflow\predict_videos.py", line 270, in analyze_videos
>>>     sess, inputs, outputs = predict.setup_GPUpose_prediction(dlc_cfg,allow_growth=allow_growth)
>>>
>>>   File "C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\predict.py", line 213, in setup_GPUpose_prediction
>>>     restorer = tf.compat.v1.train.Saver()
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by jeylau** [comment]
Hi @koronfelina, thanks for raising this issue.
In the terminal (the place where you activate your DeepLabCut environment), could you run `ipython`, and then:
```python
import deeplabcut

print(deeplabcut.__version__)
```
This would give you the installed version, which is very helpful to us 🙌

**Comment by MMathisLab** [comment]
seems the weights are no longer downloaded; are you running this in another place than where you trained originally? 

```
NOT_FOUND: Key resnet_v1_101/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
         [[node save/RestoreV2
 (defined at C:\Users\81904\anaconda3\envs\deeplabcut\lib\site-packages\deeplabcut\pose_estimation_tensorflow\core\predict.py:213)
]]
```

**Comment by koronfelina** [comment]
Hi Yes that worked. The version is 2.2.0.2. Thank you

Lina

**Comment by koronfelina** [comment]
Yes, the deeplabcut files and training on sample videos was done on an internal drive while the videos I analyze are saved in an external drive (I analyze many videos that were not used for the training). It wasn’t a problem when I did the initial training (~7 times). Now that I retrained the network one more time I get this problem and can’t evaluate the network or analyze anything in the external drive as I did before this last re-training.
